apb/templates/broker-user-auth.yaml

```diff
@@ -0,0 +1,17 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: asb-user-access
+  labels:
+    {% if broker_sandbox_role == 'admin' %}
```

Comment:
I think it needs to be formatted more like:

```
labels:
{% if broker_sandbox_role == 'admin' %}
    rbac.authorization.k8s.io/aggregate-to-admin: "true"
{% endif %}
...
```

Otherwise, I think the whitespace is included.